css-fonts/styles.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: merriweather, roboto condensed;
+}
```

Comment:
Make sure the page is displaying both fonts somewhere. Right now, `roboto condensed` is only acting as a fallback if `merriweather` fails to load.